docs/tutorials/scalar.md

```
@@ -90,6 +91,27 @@ Some options are available to configure Scalar, Ts.ED and the default spec infor
 | pathPatterns         | `[]`                                                          | Include only controllers whose paths match the pattern list provided.                                 |
 | sortPaths            | `true`                                                        | Sort paths by alphabetical order.                                                                     |
 
+:::tip
+Ts.ED relies on the latest version of the @scalar/api-reference package. If you need to use a specific version or have encountered an issue, you can specify the required version:
+```typescript
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)